pkg/globalnet/controllers/ipam/gatewaymonitor.go

```diff
@@ -266,3 +267,19 @@ func configureTCPMTUProbe() {
 		klog.Warningf(err.Error())
 	}
 }
+
+func clearGlobalnetChains(ipt iptables.Interface) {
+	klog.Info("Globalnet is enabled and active gateway migrated, flushing Globalnet chains.")
```

Comment:
```suggestion
	klog.Info("Active gateway migrated, flushing Globalnet chains.")
```

As this is now called from Globalnet code itself.

Reply:
Done.